tests/test_cli.py

```diff
@@ -32,3 +33,4 @@ def test_view_path_is_dir():
 
         # Close all existing viewer instances to avoid leaking of viewers between tests
         Viewer.close_all()
+        shutil.rmtree(f)
```

Comment:
Do we need this? isn't this deleted by default? In general we try to avoid to use `shutils` for safety reasons

Reply:
This was a remnant from having 2 distinct tests both writing a `SpatialData` object to the same location. Given the issue with overwrite this caused the tests to fail. However since then one of these tests is not there anymore so it is currently not necessary. Resolved with ([7ebcce5](https://github.com/scverse/napari-spatialdata/pull/120/commits/7ebcce54d0ff378cad824088c8cebbe18af645dd)